### PR TITLE
some first notes towards the advanced git/github session

### PR DESCRIPTION
**Pull request by nevrome**
### PR Checklist:

* [ ] Have you rendered and checked the website after your modifications with [docsify-cli](https://docsify.js.org/#/quickstart) ?
* [ ] Have you updated the `docs/_sidebar.md` file if you added new markdown files ?

### CODE REVIEW

File: day2/advanced_git.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+- Profile page: Your personal page and what you can learn from it.
+- The repository: The main unit of interaction and how it works.
+
+## Hands-on experience
+
+### Setup a Github repository and connect to it
+
+- Create an own repository: Your Name + your favorite animal -> ClemensDonkey
+- Connect to [github via ssh](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh):
+	- Generating a new SSH key
```
Comment (maxibor) [inline day2/advanced_git.md]: Add `ssh-keygen`

**Review by maxibor — APPROVED** [review]

File: day2/advanced_git.md (hunk 1 above)
Comment (maxibor) [reply day2/advanced_git.md]: can you mention the command here @nevrome ?

Comment (nevrome) [reply day2/advanced_git.md]: done

**Comment by maxibor** [comment]
You will need to sync with the master branch before we can merge @nevrome

**Pull request merged**